sip-servlets-impl/src/main/java/org/mobicents/servlet/sip/message/SipServletRequestImpl.java

```diff
@@ -237,7 +237,7 @@ public ModifiableRule getModifiableRule(String headerName) {
 			}
 		}
 		
-		if(hName.equals(ContactHeader.NAME)) {
+		if(hName.equalsIgnoreCase(ContactHeader.NAME)) {
```

Comment:
it seems the rest of headers were already using IgnoreCase, so it seems correct, lets merge it